apps/trading/components/bootstrapper/data-loader.tsx

```diff
@@ -29,6 +30,9 @@ export const DataLoader = ({
     loading: loadingAssets,
   } = useAssetsMapProvider();
 
+  useSuspenseAssets();
```

Comment:
It bugs me that the whole app will be blocked by two requests for markets and two for assets. Can we move the suspense queries to the root layout of the amm pages only.